core/kits/documents/kit.php

```diff
@@ -21,7 +21,7 @@ public function __construct( array $data = [] ) {
 		$this->tabs = [
 			'site_identity' => new Tabs\Site_Identity( $this ),
 			'lightbox' => new Tabs\Lightbox( $this ),
-			'colors_and_typography' => new Tabs\Colors_And_Typography( $this ),
+			'global_style' => new Tabs\Global_Style( $this ),
```

Comment:
The panel page title is "Colors and Typography" so, Please rename all related code to this term.